Implemented CTRL+S to save, per

JansenSmith opened the pull request:
https://github.com/CommonWealthRobotics/BowlerStudio/issues/284

Created method in scripting widget to save currently selected file. Set event listener to listen for CTRL+S and to use method.

Commit:
Implemented CTRL+S to save, per
https://github.com/CommonWealthRobotics/BowlerStudio/issues/284

madhephaestus commented on src/main/java/com/neuronrobotics/bowlerstudio/tabs/LocalFileScriptTab.java:
```diff
@@ -290,6 +290,23 @@ public void mouseClicked(MouseEvent e) {
+			/**
+			 * 
+			 */
+			private static final long serialVersionUID = -4698223073831405851L;
```
change this to a number different than the other event

Commit:
Changed serialVersionUID for CTRL+S to a random new value, per
https://github.com/CommonWealthRobotics/BowlerStudio/pull/285#discussion_r967497681
